src/Microsoft.DotNet.Build.Tasks.Packaging/src/PackageFiles/PackageLibs.targets

```diff
@@ -73,18 +73,20 @@
       <XmlDocFile Remove="@(XmlDocFile)" Condition="!Exists('%(XmlDocFile.Identity)')"/>
     </ItemGroup>
 
-    <!-- Desktop libraries that contain types in classic assemblies need to be included in ref
-         so that the types unify with the classic assemblies from the targeting pack. -->
     <PropertyGroup>
-      <PackageAsRefAndLib Condition="'$(PackageAsRefAndLib)' == '' AND '$(IsPartialFacadeAssembly)' == 'true' AND $(PackageTargetFramework.StartsWith('net4', StringComparison.OrdinalIgnoreCase))">true</PackageAsRefAndLib>
+      <IsDesktopLibrary Condition="'$(IsDesktopLibrary)' == '' AND $(PackageTargetFramework.StartsWith('net4', StringComparison.OrdinalIgnoreCase))">true</IsDesktopLibrary>
```

Comment:
From a naming prospective I think we should call this IsNetFxLibrary instead of IsDesktopLibrary.